NutritionManagement/src/calculation/NutritionCalculator.java

```diff
@@ -26,13 +33,48 @@ public static NutritionValue accumulateNutritionValue(ArrayList<NutritionValue>
      */
     public static NutritionValue computeAverage(ArrayList<Day> days) {
         return new NutritionValue(
-                days.stream().map(day -> day.getTotalNutrition()).mapToDouble(item -> item.getCalories()).average()
+                days.stream().map(day -> day.getTotalNutrition())
+                        .mapToDouble(item -> item.getCalories())
+                        .average()
                         .orElse(0),
-                days.stream().map(day -> day.getTotalNutrition()).mapToDouble(item -> item.getProtein()).average()
+                days.stream().map(day -> day.getTotalNutrition()).mapToDouble(item -> item.getProtein())
+                        .average()
                         .orElse(0),
-                days.stream().map(day -> day.getTotalNutrition()).mapToDouble(item -> item.getCarbs()).average()
+                days.stream().map(day -> day.getTotalNutrition()).mapToDouble(item -> item.getCarbs())
+                        .average()
                         .orElse(0),
-                days.stream().map(day -> day.getTotalNutrition()).mapToDouble(item -> item.getFat()).average()
+                days.stream().map(day -> day.getTotalNutrition()).mapToDouble(item -> item.getFat())
+                        .average()
                         .orElse(0));
     }
+
+    /**
+     * Input total NutritionValue of a Day and returns a new NutritionValue object,
+     * each field is corresponding to the percentage of the field in recommended
+     * NutritionValue,
+     * represented in double (0.12 => 12%)
+     * 
+     * @param nutritionValue
+     * @return
+     */
+    public static NutritionValue computePercentage(NutritionValue nutritionValue) {
```

Comment:
The `computePercentage` method correctly computes the percentage values, but consider handling the case where the reference value is zero to avoid division by zero errors.